### PR TITLE
Dask gateway bump

### PR DESCRIPTION
**Pull request by TomAugspurger**
cc @scottyhq and @tjcrone. Haven't updated the aws or OOI configs yet.

TODO:

- [ ] bump image used on icesat2 (cc @scottyhq)
- [ ] ensure ooi has dask-gateway 0.7.1 (cc @tjcrone)
- [ ] find replacement for timeouts: https://github.com/pangeo-data/pangeo-cloud-federation/pull/586/files#diff-7dd030b1241767a1e0ed7818a6e83ee4R49

### CODE REVIEW

**Comment by scottyhq** [comment]
Thanks @TomAugspurger - I just tagged 2020.04.22 images with dask-gateway 0.7.1 if you want to point to them https://github.com/pangeo-data/pangeo-docker-images/releases

**Comment by TomAugspurger** [comment]
I think I'm going to old off on using the new docker images for a separate PR, trying to keep this focused just on the dask-gateway bump. We do some stuff in postBuild that might need a replacement.

**Comment by TomAugspurger** [comment]
Pushing up changes for the other deployments shortly.

File: deployments/dev/image/binder/Dockerfile [hunk 1]
```diff
@@ -1,2 +1 @@
-# Note that there must be a tag
-FROM pangeo/pangeo-notebook-onbuild:2019.12.06
+FROM base-notebook:2020.04.22
```
Comment (scottyhq) [inline deployments/dev/image/binder/Dockerfile]: `FROM pangeo/base-notebook:2020.04.22`

**Review by scottyhq — APPROVED** [review]
Happy to have this merged any time @TomAugspurger

**Comment by TomAugspurger** [comment]
Yeah, I think we can try merge this to try it out for staging any time.

**Comment by tjcrone** [comment]
Hi sorry I have been mostly offline this week. Is there something I can help with? I see that this was merged but the OOI charts do not look like they were redeployed.

**Comment by TomAugspurger** [comment]
No worries. I think just the command in https://github.com/pangeo-data/pangeo-cloud-federation/pull/589

```
kubectl apply -f https://raw.githubusercontent.com/dask/dask-gateway/0.7.1/resources/helm/dask-gateway/crds/traefik.yaml -n staging
```

**Pull request merged**